### PR TITLE
Refactoring 

### PR DESCRIPTION
**Pull request by katalyzator**
- Changed URL structure
- Removed duplicated serializers
- Added API docs
- Changed import part

### CODE REVIEW

File: backend/requirements.txt [hunk 1]
```diff
@@ -16,3 +16,5 @@ requests==2.23.0
 six==1.14.0
 sqlparse==0.3.1
 urllib3==1.25.8
+coreapi==2.3.3
```
Comment (BupycHuk) [inline backend/requirements.txt:19]: А это точно тут используется?

Comment (katalyzator) [reply backend/requirements.txt:19]: Да, для документации

**Comment by isabek** [comment]
Good job @katalyzator

File: backend/distributor/serializers.py [hunk 2]
```diff
@@ -52,13 +56,13 @@ class Meta:
...
     phone_numbers = HospitalPhoneNumberSerializer(many=True, read_only=True)
-    statistics = StatisticPhoneNumberSerializer(many=True, read_only=True)
     location = serializers.SerializerMethodField()
+    statistics = StatisticPhoneNumberSerializer(many=True, read_only=True)
     needs = HospitalNeedsSerializer(many=True, read_only=True)
 
     class Meta:
         model = Hospital
-        fields = ['id', 'name', 'code', 'address', 'location', 'locality_id', 'phone_numbers', 'statistics', 'needs']
+        fields = ('id', 'name', 'code', 'address', 'location', 'phone_numbers', 'needs', 'locality_id', 'statistics')
```
Comment (isabek) [inline backend/distributor/serializers.py]: Let's put locality_id after location.

Comment (katalyzator) [reply backend/distributor/serializers.py]: Is the location of the fields important?

Comment (isabek) [reply backend/distributor/serializers.py]: :) frontend guys then will ask

Comment (katalyzator) [reply backend/distributor/serializers.py]: Done

**Pull request merged**